docs/source/features/passes/onnx.md

```diff
@@ -79,6 +79,103 @@ Please refer to [OrtTransformersOptimization](ort_transformers_optimization) for
 }
 ```
 
+`AppendPrePostProcessingOps` also support customize pre and post processing ops by leveraging the [onnxruntime-extension steps](https://github.com/microsoft/onnxruntime-extensions/tree/main/onnxruntime_extensions/tools/pre_post_processing/steps) and `PrePostProcessor`.
+You can refer to [here](https://github.com/microsoft/onnxruntime-extensions/blob/main/onnxruntime_extensions/tools/Example%20usage%20of%20the%20PrePostProcessor.md) to see how to leverage `PrePostProcessor` to customize pre and post processing ops.
+
+Here are some examples to describe the pre/post processing which is exactly same with [superresolution](https://github.com/microsoft/onnxruntime-extensions/blob/main/onnxruntime_extensions/tools/add_pre_post_processing_to_model.py#L89)
+
+```json
+{
+    "pre": [
+        {"ConvertImageToBGR": {}},
+        {
+            "Resize": {
```

Comment:
Is there a central doc/list where we can find the pre/post ops list from onnxruntime-extensions? The examples are helpful, but how we guide user to complete the pre/post configs?

Reply:
I can only find the full list by onnxruntime extension repo. And  there is no any document for the full list.